titiler/pgstac/href_exchange.py

```diff
@@ -0,0 +1,29 @@
+import copy
+import logging
+import requests
+from titiler.pgstac.settings import HrefExchangeSettings
+href_exchange_settings = HrefExchangeSettings()
+from urllib.parse import urljoin
+
+
```

Comment:
is there stuff in here you could use? https://pystac.readthedocs.io/en/0.4/api.html#pystac.Item.normalize_hrefs